doc-content/enterprise-only/cases/case-management-case-key-prefix-proc.adoc

```diff
@@ -0,0 +1,86 @@
+[id='case-management-case-key-prefix-proc-{context}']
```

Comment:
In your anchors, the format is proc_{context}. You can change later.

Reply:
Also, this isn't a procedure module. It's sort of a concept module with implied actions. I would rewrite as a true procedure module. Again, you can do it after release. We can have a meeting if  you like.

Reply:
I agree. Started out procedural, then changed direction. I will make a note to meet post GA to go through this entire doc. Thank you